docs/configuration/readme.md

```diff
@@ -675,15 +671,18 @@ Refresh cooldown is the minimum amount of time between allowed manually refreshe
 Default Upstream Timeout is the default timeout applied to a proxied route when no `timeout` key is specified by the policy.
 
 ### JWT Claim Headers
+
 - Environmental Variable: `JWT_CLAIMS_HEADERS`
 - Config File Key: `jwt_claims_headers`
-- Type: `string list`
-- Example: `email,groups`, `user`
+- Type: slice of `string`
+- Example: `email`,`groups`, `user`
 - Optional
-  
-Set this option for the pomerium proxy to copy JWT claim information into request headers with the name `x-pomerium-claim-*`.  Any claim listed in the pomerium JWT can be placed into a corresponding header for downstream consumption.  This claim information is sourced from your IDP and pomerium's own session metadata.
 
-Use this option if you previously relied on `x-pomerium-authenticated-user-{email|user-id|groups}` for downstream authN/Z.  
+The JWT Claim Headers setting allows you to pass specific user session data down to downstream applications as HTTP request headers. Note, unlike the header `x-pomerium-jwt-assertion` these values are not signed by the authorization service.
+
+Any claim in the pomerium session JWT can be placed into a corresponding header for downstream consumption. This claim information is sourced from your IDP and pomerium's own session metadata.
```

Comment:
It is always fun, reviewing your PR because of the humour. I think on line `683` we can further improve on it by making the reader aware of what IDP means except we want to assume our audience knows.

How about `".....from your Identity Provider (IDP) ..."` or just using finding somewhere at the beginning of the page to introduce `Identity Provider (IDP)`, since we tend to use `IDP` in a lot of places, what do you think?

Reply:
Good idea.